README.md

```diff
@@ -24,4 +24,4 @@ For complete documentation, see [Dart gRPC](https://grpc.io/docs/languages/dart)
 
 If you experience problems or have feature requests, [open an issue](https://github.com/dart-lang/grpc-dart/issues/new).
 
-Note that we have limited bandwidth to accept PRs, and that all PRs require signing the [EasyCLA](https://lfcla.com).
+Note that we have limited bandwidth to accept PRs, and that all PRs require signing the [EasyCLA](https://lfcla.com).
```

Comment:
Seems to be an accidental edit (though I can't clearly see what the edit was from GitHub UI. Maybe newline?). Consider fully reverting by doing 

```
git checkout origin/master -- README.md
```

Reply:
Ah - I had removed the empty carriage return at the end of the file, so putting that back fixed it back to how it was. Fixed in https://github.com/bottlepay/grpc-dart/commit/b8ffd5059d34e36f5e0ece961b6ac5e852955182.

Reply:
Fixed comment for `decodeStatusDetails` in https://github.com/bottlepay/grpc-dart/commit/150268e6e7766db7ae324ae82fa64b73034532ff